README.md

```diff
@@ -52,6 +52,26 @@ This is the preferred way of handling JSON requests.
 ```
 
 
+##### Transforming JSON map keys
```

Comment:
Can you make this section more concise? So something like:

    Both `wrap-json-response` and `wrap-json-body` support the `:key-fn` option
    which takes a transform function in order to customize the JSON map keys:

    ```clojure
    (wrap-json-body handler {:key-fn clojure.string/lower-case})
    ```

Also remove the header, please.